v2/stacks/dataset-catalogue/provisioning/mongodb/bundle-events.json

```
@@ -1,243 +1,160 @@
-          "preview": "https://publishing.ons.gov.uk/data-admin/preview/datasets/dataset-3/editions/march/versions/1"
-        }
-      }
-    }
-    {
-      "created_at": "2025-05-20T10:45:12.321Z",
-      "requested_by": {
-        "id": "0889d599-3f0e-4564-9d6e-9455a6b73da7",
-        "email": "publisher@ons.gov.uk"
-      },
-      "action": "CREATE",
-      "resource": "/bundles",
-      "data": {
-        "content_type": "DATASET",
-        "metadata": {
-          "dataset_id": "dataset-4",
-          "edition_id": "april",
-          "version_id": 1,
-          "title": "Dataset 4"
-        },
-        "id": "b5c4d3e2-f1g0-h9i8-j7k6-l5m4n3o2p1q0",
-        "bundle_id": "bundle-4",
-        "state": "PUBLISHED",
-        "links": {
-          "edit": "https://publishing.ons.gov.uk/data-admin/edit/datasets/dataset-4/editions/april/versions/1",
-          "preview": "https://publishing.ons.gov.uk/data-admin/preview/datasets/dataset-4/editions/april/versions/1"
-        }
-      }
-    }
-    {
-      "created_at": "2025-05-22T15:33:27.654Z",
-      "requested_by": {
-        "id": "0889d599-3f0e-4564-9d6e-9455a6b73da7",
-        "email": "publisher@ons.gov.uk"
-      },
-      "action": "CREATE",
-      "resource": "/bundles",
-      "data": {
-        "content_type": "DATASET",
-        "metadata": {
-          "dataset_id": "dataset-5",
-          "edition_id": "may",
-          "version_id": 1,
-          "title": "Dataset 5"
-        },
-        "id": "w3x4y5z6-a7b8-c9d0-e1f2-g3h4i5j6k7l8",
-        "bundle_id": "bundle-5",
-        "state": "DRAFT",
-        "links": {
-          "edit": "https://publishing.ons.gov.uk/data-admin/edit/datasets/dataset-5/editions/may/versions/1",
-          "preview": "https://publishing.ons.gov.uk/data-admin/preview/datasets/dataset-5/editions/may/versions/1"
-        }
-      }
-    }
-    {
-      "created_at": "2025-05-22T15:40:58.987Z",
-      "requested_by": {
-        "id": "0889d599-3f0e-4564-9d6e-9455a6b73da7",
-        "email": "publisher@ons.gov.uk"
-      },
-      "action": "CREATE",
-      "resource": "/bundles",
-      "data": {
-        "bundle_type": "MANUAL",
-        "preview_teams": [
-          { "id": "3c4d5e6f-7g8h-9i0j-1k2l-3m4n5o6p7q8r" }
-        ],
-        "title": "Bundle 5",
-        "state": "DRAFT",
-        "id": "c6d7e8f9-0a1b-2c3d-4e5f-6g7h8i9j0k1l",
-        "created_by": {
-          "email": "publisher@ons.gov.uk"
-        },
-        "created_at": "2025-05-22T15:40:58.987Z",
-        "managed_by": "DATA-ADMIN"
-      }
-    }
\ No newline at end of file
+{
+  "created_at": "2025-05-15T09:30:42.111Z",
+  "requested_by": {
+    "id": "0889d599-3f0e-4564-9d6e-9455a6b73da7",
+    "email": "publisher@ons.gov.uk"
+  },
+  "action": "CREATE",
+  "resource": "/bundles/bundle-1",
+  "data": {
+    "dataset_id": "dataset-1",
+    "edition_id": "january",
+    "item_id": "bundle-1",
+    "state": "draft",
+    "url_path": "datasets/dataset-1/editions/january/versions/1"
```

```suggestion
    "url_path": "/datasets/dataset-1/editions/january/versions/1"
```
Since the paths are relative they need to start with a `/`